mitodl/mitxpro
# Fix various build/run issues

#### What are the relevant tickets?
N/A

#### What's this PR do?
Fixes a few build issues

#### How should this be manually tested?
- Run `docker-compose build --no-cache`
- Run `docker-compose up` , the app should start
